anomaly-benchmarking/make-binary/readme.md

```diff
@@ -0,0 +1,41 @@
+# Making a Dataset Binary
+
+This library takes any dataset and returns it extended by a new field,
+"Is Anomaly?". "Is Anomaly?" takes the values "normal" or "anomalous"
```

Comment:
I changed the name to "instance type". The label is so arbitrary (what is "normal" and "anomalous" depends, among other things, on the order of the rows) it feels too much like lying to call it "anomaly?"/"true".